game/src/main/java/tw/waterball/survive/io/game/gameloop/GameLoop.java

```diff
@@ -0,0 +1,43 @@
+package tw.waterball.survive.io.game.gameloop;
+
+import tw.waterball.survive.io.game.sprite.Sprite;
+import tw.waterball.survive.io.game.world.SpriteContainer;
+
+/**
+ * @author - a89010531111@gmail.com
+ */
+public class GameLoop {
+    private final SpriteContainer spriteContainer;
+    private final GameListener gameListener;
+    private boolean running = false;
+
+    public GameLoop(SpriteContainer spriteContainer, GameListener gameListener) {
```

Comment:
No `GameListener` as the parameter, since the listeners will be added by a adder `addListener`